keyboards/kprepublic/bm40hsrgb/readme.md

```diff
@@ -12,10 +12,10 @@ This firmware was originally taken from a kbfirmware.json and manually converted
 
 Make example for this keyboard (after setting up your build environment):
 
-    make bm40hsrgb:default
+    make kprepublic/bm40hsrgb:default
```

Comment:
```suggestion
    qmk compile -kb kprepublic/bm40hsrgb -km default
```
Perhaps you can also take this opportunity to update the build command documentation to use the `qmk` python script that is agnostic to current folder.